src/theme/buttons/css/base.css

```diff
@@ -65,6 +65,7 @@
 
   &:disabled {
     opacity: 0.58;
+    cursor: default;
```

Comment:
```suggestion
    cursor: not-allowed;
```

Reply:
I thought of that one but `cursor: default;` seems to be the current state for disabled buttons when i searched through the codebase, I didn't want to break any current design patterns (I have worked with designers that think `not-allowed` just adds confusion). Many "go to" design systems dont use it either and it is not even used here on github. 

Personally, as a user it would just add confusion to me because barely no other site is using it for disabled states.

Reply:
Hmm i don't agree with you at all :)

I think we should make use as much as possible of cursor interaction — is a very important accessibility aid for users, specially the ones using keyboard controls.

That Github or other websites don't use them does not mean they are right :)

[MDN](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/CSS_and_JavaScript) explains good the point I'm addressing on this comment.

`cursor: not-allowed`

Reply:
I am fine with using it as long as it is consistent throughout the application. I have created an issue regarding this https://github.com/LiskHQ/lisk-desktop/issues/4664